File: src/main/java/de/tuberlin/dima/flink/model/StudentJobs.java
package de.tuberlin.dima.flink.model;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by mustafa on 18/05/15.
 */
public class StudentJobs {

	private String name;
	private List<String> jobs = new ArrayList<String>();

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public List<String> getJobs() {
		return jobs;
	}

	public void setJobs(List<String> jobs) {
		this.jobs = jobs;
	}

	@Override
	public String toString() {
		StringBuffer buffer = new StringBuffer();
		buffer.append(name).append(";");

		int counter = 0;
		for(String str : jobs) {
			buffer.append(str);
			if(counter < jobs.size()-1) {
				buffer.append(";");
				counter++;
			}
		}
		return buffer.toString();
	}
}

File: src/main/scala/de/tuberlin/dima/model/StudentJobs.scala
package de.tuberlin.dima.model

import scala.collection.mutable.ArrayBuffer

class StudentJobs {

	private var _name: String = _
	private val _jobs: ArrayBuffer[String] = new ArrayBuffer[String]

	def name = _name
	def name(s:String):Unit = {_name = s}

	def jobs = _jobs
	def jobs(arrayBuffer: ArrayBuffer[String]): Unit = {
		_jobs ++=(arrayBuffer)
	}

	override def toString: String = {
		_name + " [" + _jobs.foreach(print(_)+";") + "]"
	}

}